active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,393 @@
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="auto"/>
+        </Grid.RowDefinitions>
+        <controls:DataGrid x:Name="dataGrid" AutoGenerateColumns="True"/>
+        <muxc:ProgressRing x:Name="progressRing" Visibility="Collapsed" IsActive="false">
+        <muxc:Pager x:Name="pager" Grid.Row="1" PageChanged="OnPageChanged" PagerDisplayMode="NumberPanel"/>
+    </Grid>
+</Page>
+```
+C#
+```CSharp
+public sealed partial class DataGridSamplePage : Page
+{
+        private const int imagesPerPage = 25;
+
+        public DataGridSamplePage()
+        {
+                this.InitializeComponent();
+        }
+
+        private async void OnPageLoaded(object sender, RoutedEventArgs e)
+        {
+                pager.NumberOfPages = await ImageGenerator.GetNumberOfPages(imagesPerPage);
+                dataGrid.ItemsSource = await ImageGenerator.GetImagesInPage(0, imagesPerPage);
+                progressRing.Visibility = "Collapsed";
+                progressRing.IsActive = false;
+        }
+        private async void OnPageChanged(Pager sender, PageChangedEventArgs args)
+        {
+                dataGrid.ItemsSource = await ImageGenerator.GetImagesInPage(args.CurrentPage, imagesPerPage);
+                progressRing.Visibility = "Visible";
+                progressRing.IsActive = true; 
+        }
+}
+```
+
+# API Notes
+| Name | Description| Default | 
+|:---:|:---|:---|
+| PagerDisplayMode | Enum that contains 4 values (Auto, ComboBox, NumberBox, ButtonPanel) to represent the look of the pager control. When Auto is selected, the display mode will be ComboBox if the NumberOfPages property is less than 11 otherwise it will be NumberBox. | Auto |
+| PagerButtonVisibilityBehavior | Enum that contains 4 values (Visible, HiddenOnEdge, Hidden) that allows the app developer to hide or show the four edge buttons. HiddenOnEdge will remove the appropriate buttons if the selected page is the last or first page. When the last page is selected, the next and last buttons will be disabled and same when the first page is selected for the first and previous page. When Auto is selected, the visibility mode will be AlwaysVisible. | Visible |
+| NumberOfPages | Sets the max number of pages the index control will iterate through. The default will represent an infinite page range. | -1 |
+| First, Previous, Next, and Last ButtonCommand | Specially handle the button pressed event for when the end user selects the buttons. | N/A |
+| First, Previous, Next, and Last Style | Give the developer the option to customize the style by changing the text or glyph for the edge buttons.| N/A |
+| ButtonPanelAlwaysShowFirstAndLastPage | Note: This property only applies to the button panel display mode.Boolean to display the ellipses and the first and last index of the numerical button panel display mode. | True |
+| SelectedIndex | The 0 based index that is currently selected. It will default to the first index. | 0 |
```

Comment:
The IDL code says that this is SelectedPageIndex, is that correct, and this entry wasn't updated? Or should we stick with SelectedIndex?

Also, just to clarify, when using this control in ComboBox, when the user clicks on entry "2", the PagerControl will report it's SelectedPageIndex as 1 right?

Reply:
Yes that is correct! I also updated the name so it will say SelectedPageIndex.